layers/pct.py

```diff
@@ -0,0 +1,54 @@
+from utils import *
+
+
+def makePct():
+    tmp = "tmp_pct"
+    ensureEmptyTempDirectoryExists(tmp)
+
+    run(
+        [
+            "wget",
+            "https://github.com/npct/pct-outputs-national/raw/master/commute/lsoa/rnet_all.geojson",
+            "-O",
+            f"{tmp}/commute.geojson",
+        ]
+    )
+
+    run(
+        [
+            "wget",
+            "https://github.com/npct/pct-outputs-national/raw/master/school/lsoa/rnet_all.geojson",
+            "-O",
+            f"{tmp}/school.geojson",
+        ]
+    )
+
+    # The two trip purposes are split into different files, and neither feature
```

Comment:
@robinlovelace-ate , ideally we'd have 6 properties on each feature (the 3 scenarios for commuting and then for school). But there's no straightforward way to join the two files based on `local_id` or the feature ID. Is that correct?

Reply:
Yes, no easy way to join the 2... We can pre-process them into a single route network but I think it makes sense to keep them separate, cycling to school is a different kettle of fish.